### PR TITLE
#3805 Fill up translations strings where we are missing them

### LINKED ISSUE #3805: Fill up translations strings where we are missing them
We'll need to fill up translations where we are missing them.

Revisit 
- landing page
- explorer
- dialogues
- search bar 
- I'll probably miss other else where locations :)
- remove translations which aren't used in the app anymore
  -  remove it in particular other languages as well 🙏	

image

_Originally posted by @yangwao in https://github.com/kodadot/nft-gallery/issues/3788#issuecomment-1223872647_

### PR DESCRIPTION
**Pull request by prachi00**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).
👇 \_ Let's make a quick check before the contribution.

### PR type

- [ ] Bugfix
- [ ] Feature
- [x] Refactoring

### What's new?

- [x] PR closes #3805
- [ ] Requires deployment <rubick/magick_issue>
- [ ] <brief_description_of_what_I've_added>

### Before submitting Pull Request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [ ] I've posted a screenshot of demonstrated change in this PR

### Optional

- [ ] I've tested it at </rmrk/collection/26902bc2f7c20c546a-1FVG7>
- [ ] I've tested PR on mobile and everything seems works
- [ ] I found edge cases
- [ ] I've written some unit tests 🧪

### Had issue bounty label?

- [x] Fill up your KSM address: 
[Payout](https://beta.kodadot.xyz/transfer/?target=EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU)

### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

### Screenshot

- [ ] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

### CODE REVIEW

**Review by kkukelka — APPROVED** [review]
code lgtm

**Comment by petersopko** [comment]
@prachi00 one conflict now came up because of, please let's include strings from there into this PR as well 🙏🏻 
- #3872

File: components/common/WalletModal.vue [hunk 1]
```diff
@@ -74,7 +74,7 @@
 
         <div v-if="hasSelectedWalletProvider && hasWalletProviderExtension">
           <div class="subtitle is-size-6 has-text-centered">
-            Choose your
+            {{ $t('general.chooseWallet') }}
```
Comment (roiLeo) [inline components/common/WalletModal.vue:77]: missing `account` after wallet Image

**Review by roiLeo — COMMENTED** [review]
Hey! Could you take a look on:
- `MetadataUpload` label 
- `RampCredit` page
- some label in IdentityForm, Sales, Spotlight & Series
- default sharing label?
- Teleport & Transfer page

**Comment by DeluneClaire** [comment]
> @prachi00 one conflict now came up because of, please let's include strings from there into this PR as well 🙏🏻
> 
>     * [refactor (3733): error messaging on Creative Minting #3872](https://github.com/kodadot/nft-gallery/pull/3872)

Hello, I'm a little bit confused about what to test here... I see there are going to be some updates, but then? :/

**Comment by petersopko** [comment]
> Hello, I'm a little bit confused about what to test here... I see there are going to be some updates, but then? :/

we are changing here the source of all the text rendered on page (for easier translations). Therefore, the way to test is from the user perspective, is to look throughout the app and see if there is no weird text being rendered somewhere.

**Comment by prachi00** [comment]
> Hey! Could you take a look on:
> 
> * `MetadataUpload` label
> * `RampCredit` page
> * some label in IdentityForm, Sales, Spotlight & Series
> * default sharing label?
> * Teleport & Transfer page

done, except the sharing label part, I didnt get it, can you elaborate please?

**Comment by roiLeo** [comment]
> done, except the sharing label part, I didnt get it, can you elaborate please?

https://github.com/kodadot/nft-gallery/blob/19ca2eb158885a00afaff7fdfa6c89d72b7d2e97/components/shared/Sharing.vue#L113

I've just seen that the text "URL copied to clipboard" isn't translated

**Comment by prachi00** [comment]
> URL copied to clipboard

Screen Shot 2022-09-05 at 9 14 44 PM
 URL copied to clipboard is already in en.json

**Review by roiLeo — APPROVED** [review]
🚀

**Comment by kkukelka** [comment]
pay 100

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $100 @ 47.13 USD/KSM ~ 2.122 $KSM
🧗 [EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU](https://kusama.subscan.io/account/EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU)
🔗 [0xf5f1a482cb9cafc67cf061a4413a2c406e12c048437470db74451d55ba05fc57](https://kusama.subscan.io/extrinsic/0xf5f1a482cb9cafc67cf061a4413a2c406e12c048437470db74451d55ba05fc57)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

File: components/common/WalletModal.vue [hunk 2]
```diff
@@ -72,14 +72,18 @@
           </b-button>
         </div>
 
-        <div v-if="hasSelectedWalletProvider && hasWalletProviderExtension">
+        <div
+          v-if="
+            (hasSelectedWalletProvider && hasWalletProviderExtension) || true
```
Comment (roiLeo) [inline components/common/WalletModal.vue:77]: 😢

**Pull request merged**